api/proto/teleport/workloadidentity/v1/sigstore_policy_resource.proto

```diff
@@ -0,0 +1,112 @@
+// Copyright 2025 Gravitational, Inc
```

Comment:
This file is probably the most interesting part of the PR as the rest is just boilerplate.